.github/workflows/go.yml

```diff
@@ -15,6 +15,16 @@ env:
   golangci-lint-version: v1.46.0
 
 jobs:
+  scripts:
```

Comment:
As it came up in a recent discussion, I would argue that we should not only have this check in an action, but also as a target in our Makefile, so that it can also be tested locally.

Reply:
And if only there was a tool that people could use to have all of the dependencies installed that we need to build/check the codebase...